tests/Integration/inc/Functions/rocketCleanCacheBusting.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\Functions;
```

Comment:
To fit the plugin’s folders structure, the folder and the namespace for `Functions` should be lowercase.
```suggestion
namespace WP_Rocket\Tests\Integration\inc\functions;
```

Maybe it's a job for another PR though, since it's the same for all tests related to functions.